README.md

```diff
@@ -85,6 +85,20 @@ Edit it by adding the following lines:
         sourcelist: '{"RaspberryPi": "KEY_HDMI2", "Chromecast": "KEY_HDMI3"}'
         protocol: ctl_beta
     ```
+    ### Example configuration.yaml for ctl_beta protocol for H/J Models
+    ```
+     media_player:
+       - platform: samsungtv_custom
+         host: IP_ADDRESS
+         port: (8001 or 8002)
+         mac: MAC_ADDRESS
+         sourcelist: '{"RaspberryPi": "KEY_HDMI2", "Chromecast": "KEY_HDMI3"}'
+         protocol: ctl_beta
+         token: kdsjfbsdkjbsdkjfskdjf:2
+         id: 07270e01-0078-1000-8cd0-c4576e9dxxxxx
+    ```
+    The token get be fetched via https://github.com/eclair4151/SmartCrypto
+    The id can be fetched vie http://IP_ADDRESS:8001/api/v2/
```

Comment:
ID is empty on my UE55HU8500

Reply:
Can you please paste the response please, that you get from your TV?

Reply:
<img width="954" alt="tv" src="https://user-images.githubusercontent.com/11518567/76700597-5d636780-66b9-11ea-8d2b-2ede5c52f617.png">
For me it looks like this

Reply:
That's mine.
It actually looks weird because network connection is wired.
```json
{
  "id": "",
  "name": "(none)",
  "version": "2.0.24",
  "device": {
    "type": "Samsung SmartTV",
    "duid": "",
    "model": "Linux arm",
    "modelName": "Linux arm",
    "description": "Linux arm on ARMv7 Processor rev 3 (v7l)",
    "networkType": "wireless",
    "ssid": "",
    "ip": "192.168.1.219",
    "firmwareVersion": "3.8.13",
    "name": "(none)",
    "id": "",
    "udn": "",
    "resolution": "1920x1080",
    "countryCode": "US",
    "msfVersion": "2.0.24",
    "smartHubAgreement": "false"
  },
  "type": "Samsung SmartTV",
  "uri": "http://192.168.1.219:8001/api/v2/"
}
```